pmg/templates/committee_detail.html

```diff
@@ -139,7 +139,7 @@ <h3 class="no-results">No results.</h3>
       <table class="table table-striped">
         {% for item in recent_questions %}
           <tr class="item">
-            <td class="date"><nobr>{{ (item.start_date or item.date)|pretty_date }}</nobr></div>
+            <td class="date"><nobr>{{ (item.start_date or item.date)|pretty_date }}</nobr>
```

Comment:
Ah alright. I noticed a random `</div>` so I removed it but didn't look properly